src/main/java/com/synopsys/integration/alert/component/settings/SettingsUIConfig.java

```diff
@@ -0,0 +1,43 @@
+/**
+ * blackduck-alert
+ *
+ * Copyright (C) 2019 Black Duck Software, Inc.
+ * http://www.blackducksoftware.com/
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.synopsys.integration.alert.component.settings;
+
+import java.util.List;
+
+import org.springframework.stereotype.Component;
+
+import com.synopsys.integration.alert.common.descriptor.config.field.ConfigField;
+import com.synopsys.integration.alert.common.descriptor.config.ui.UIConfig;
+
+@Component
+public class SettingsUIConfig extends UIConfig {
+    public SettingsUIConfig() {
+        super(SettingsDescriptor.SETTINGS_LABEL, SettingsDescriptor.SETTINGS_URL, SettingsDescriptor.SETTINGS_ICON);
+    }
+
+    @Override
+    public List<ConfigField> createFields() {
+        return List.of();
```

Comment:
We might want to add a TODO here to remember to add fields in the future (If this will be auto generated)